kuma/wiki/templates/wiki/revision.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
   <article>
-    <h1>{{ document.title }}</h1>
+    <h1><a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ _('{title}')|fe(title=document.title) }}</a></h1>
```

Comment:
Is `document.title` trusted input? Or does it require `|fe` because it may contain HTML that shouldn't be escaped? Or both?

Otherwise, you should probably use `|f` instead...
```
>>> t = env.from_string('<a href="">{{ title|fe(title=title) }}</a>')
>>> t.render({'title': '<script>alert("xss")</script>'})
u'<a href=""><script>alert("xss")</script></a>'
```